spec/person_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rspec'
+require_relative '../person'
+
```

Comment:
When I run the tests I get an error.

![image](https://user-images.githubusercontent.com/38088818/234526272-f3cbb0db-3609-459f-a936-26afbae06aeb.png)

- Kindly remove the `require 'rspec'` in all the test files. There's no need to require rspec in order to run tests in your project. You simply need to add `gem 'rspec-rails', '~> 6.0.0'` to the `Gemfile`, Then run `bundle install` to install it in your project.
- Ruby will automatically require rspec in all test files ending with `*_spec.rb`.

## Summary

After I removed the `require 'rspec'`, the tests passed. Great job !!! 💯 

![image](https://user-images.githubusercontent.com/38088818/234527975-cfac4930-c08f-4185-aec6-d33f0ba0b52a.png)

Reply:
Well resolved ✔️